service-print/src/main/java/org/oskari/print/request/PrintRequest.java

```diff
@@ -131,4 +133,16 @@ public void setShowLogo(boolean showLogo) {
         this.showLogo = showLogo;
     }
 
+    public String getScaleText() {
+        return scaleText;
+    }
+
+    public void setScaleText(String scaleText) {
+        this.scaleText = scaleText;
+    }
+
+    public boolean isScaleText(){
```

Comment:
Maybe this function could have a more meaningful name? `isScaleText()` correctly explains what the function does, but the "if scaleText, force this"-part is missing

Reply:
Or maybe add a comment explaining the "why" to the `PDF#drawScale(PDPageContentStream stream, PrintRequest request)` function